scripts/lib/clean-venv-cache

```diff
@@ -23,12 +23,16 @@ def get_caches_in_use(threshold_days):
     setups_to_check = set([ZULIP_PATH, ])
     caches_in_use = set()
 
+    def add_current_venv_cache(venv_name: Text) -> None:
+        CACHE_SYMLINK = os.path.join(os.path.dirname(ZULIP_PATH), venv_name)
+        CURRENT_CACHE = os.path.dirname(os.path.realpath(CACHE_SYMLINK))
+        caches_in_use.add(CURRENT_CACHE)
+
     if ENV == "prod":
         setups_to_check |= get_recent_deployments(threshold_days)
```

Comment:
You also need to do something here to make sure it checks the `thumbor` venvs in production.

@HarshitOnGitHub may be able to provide some advice.

Reply:
We don't actually need to do anything else for handling production cache cleanup for thumbor. It is already been handled correctly. `get_recent_deployments` gets all the deployments dirs present which we want to keep or whatever and then we append to `caches_in_use` using all these dirs. Specifically the hashes of all the files in requirements folder are added to `caches_in_use` in use for all the deployments. So we don't need to do anything special here.

Development was a different case because in general we for development we after cache clean up is run we want to keep only two types of caches:
* Cache which belongs to deps present in the current branch.
* Cache present due to the last provisioning which happened which may or may not be from the same branch.

Reply:
Ahh, OK, I'd missed that we were hashing all the requirements files in that directory.  OK.